### PR TITLE
Fix commission history view for all validators

### LINKED ISSUE #5544: Commission change history only visible for logged in validator
### Expected behavior
The commission change history should be visible for all validators

### Actual behavior
Commission change history is only visible for logged in validators
Screenshot 2023-12-08 at 16 31 06
Screenshot 2023-12-08 at 16 31 39

### Steps to reproduce
- Open the validators page
- Select a validator to view their details

### Which version(s) does this affect? (Environment, OS, etc...)
All

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #5544

### How was it solved?

- Updating the logic for the commission history view

### How was it tested?

- Unit tests
- Manually

### CODE REVIEW

File: src/modules/pos/validator/components/ValidatorProfile/DetailsView.js [hunk 1]
```diff
@@ -59,9 +59,9 @@ const DetailsView = ({ data, isMyProfile }) => {
       onEdit: !isMyProfile
         ? undefined
         : () => addSearchParamsToUrl(history, { modal: 'changeCommission' }),
-      onView: !isMyProfile
-        ? undefined
-        : () => addSearchParamsToUrl(history, { modal: 'commissionHistory' }),
+      onView: commission
+        ? () => addSearchParamsToUrl(history, { modal: 'commissionHistory' })
+        : undefined,
```
Comment (ManuGowda) [inline src/modules/pos/validator/components/ValidatorProfile/DetailsView.js]: This wont work.. all validators will have 100% commission to start with.

We should not show the icon when it doesn't have history. So best it to handle them here with 

```
onView: commission < 10000
        ? () => addSearchParamsToUrl(history, { modal: 'commissionHistory' })
        : undefined,
```

**Review by ManuGowda — DISMISSED** [review]

File: src/modules/pos/validator/components/ValidatorProfile/DetailsView.js (hunk 1 above)
Comment (ikem-legend) [reply src/modules/pos/validator/components/ValidatorProfile/DetailsView.js]: Fixed

File: src/modules/pos/validator/components/ValidatorProfile/DetailsView.js [hunk 2]
```diff
@@ -59,9 +59,10 @@ const DetailsView = ({ data, isMyProfile }) => {
       onEdit: !isMyProfile
         ? undefined
         : () => addSearchParamsToUrl(history, { modal: 'changeCommission' }),
-      onView: !isMyProfile
-        ? undefined
-        : () => addSearchParamsToUrl(history, { modal: 'commissionHistory' }),
+      onView:
+        commission < 10000
```
Comment (eniolam1000752) [inline src/modules/pos/validator/components/ValidatorProfile/DetailsView.js]: would shouldn't use magic number here. We can define 10000 in a well defined constant and then re-consum here. It kind of helps readability.

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

File: src/modules/pos/validator/components/ValidatorProfile/DetailsView.js (hunk 2 above)
Comment (ManuGowda) [reply src/modules/pos/validator/components/ValidatorProfile/DetailsView.js]: @ikem-legend you can use this `MAX_VALIDATOR_COMMISSION` src/modules/pos/validator/consts/validators.js

**Review by ManuGowda — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**